### PR TITLE
Add `mc support callhome get` command

### PR DESCRIPTION
**Pull request by anjalshireesh**
Will print all the callhome settings (currently just `logs=on|off`)

### CODE REVIEW

File: cmd/support-callhome-get.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+// JSON jsonified service status Message message.
+func (m callhomeGetMessage) JSON() string {
+	statusJSONBytes, e := json.MarshalIndent(m, "", " ")
+	fatalIf(probe.NewError(e), "Unable to marshal into JSON.")
+
+	return string(statusJSONBytes)
+}
+
+func retrieveCallhomeSetting(alias string, s string) (v string, e error) {
```
Comment (vadmeste) [inline cmd/support-callhome-get.go:92]: Let's not return any value here if we are not going to use them, it is better to call this function printCallhomeSetting as well.

File: cmd/support-callhome-get.go [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+		ep, found := kvs.Lookup("endpoint")
+
+		if found && len(ep) > 0 {
+			// subnet webhook is configured. check if it is enabled
+			enable, found := kvs.Lookup("enable")
+
+			if found {
+				val = enable
+			} else {
+				// if the 'enable' key is not found, it means that the webhook is enabled
```
Comment (vadmeste) [inline cmd/support-callhome-get.go:111]: :+1:

**Review by vadmeste — COMMENTED** [review]
Few comments

File: cmd/support-callhome-get.go (hunk 1 above)
Comment (anjalshireesh) [reply cmd/support-callhome-get.go]: Thanks, done.

**Review by vadmeste — APPROVED** [review]
LGTM

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**